pkg/dnservice/cfg.go

```diff
@@ -120,5 +136,26 @@ func (c *Config) Validate() error {
 	if c.LogService.ConnectTimeout.Duration == 0 {
 		c.LogService.ConnectTimeout.Duration = defaultConnectTimeout
 	}
+	if c.Ckp.ScannerInterval == 0 {
+		c.Ckp.ScannerInterval = defaultScannerInterval
+	}
+	if c.Ckp.ExecutionInterval == 0 {
```

Comment:
set values in `dn.toml` like this

```toml
[dn.Ckp]
scanner-interval = "100s"
execution-levels = 200
execution-interval = "100s"
catalog-ckp-interval = "100s"
catalog-unckp-limit = 100
flush-interval = "100s"
```

the right value is default value

this params are needed because we want to control flushing in early bvt test